packages/manager/modules/pci/src/projects/project/kubernetes/details/service/service.controller.js

```diff
@@ -37,7 +40,14 @@ export default class KubernetesServiceCtrl {
   }
 
   downloadConfigFile() {
-    // Set yml extension manually as there is no MIME type yet
-    this.CucControllerHelper.constructor.downloadContent({ fileContent: this.kubernetesConfig.content, fileName: `${this.kubernetesConfig.fileName}.yml` });
+    this.loadingKubeConfig = true;
+    return this.getKubeConfig()
+      .then((config) => {
+        // Set yml extension manually as there is no MIME type yet
+        this.CucControllerHelper.constructor
+          .downloadContent({ fileContent: config.content, fileName: `${config.fileName}.yml` });
+      })
+      .catch(error => this.CucServiceHelper.errorHandler('kube_service_file_error')(error))
```

Comment:
We should probbaly uniformize error handling by using `CucCloudMessage` everywhere. That way if we want to change we only have to worry about one migration :)